src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String TO_BUY = "buy";
+    private static final String TO_SUPPLY = "supply";
+    private static final String RESULT = "result";
+    private static final int DATA_INDEX_TO_DO = 0;
+    private static final int DATA_INDEX_AMOUNT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = createAReport(fromFileName);
+        writeAReportToFile(toFileName, report);
```

Comment:
better use this template to refactor your code a little:
```suggestion
        String dataFromFile = readFile(fromFileName);
String report = createReport(dataFromFile);
writeReportToFile(report, toFileName);
```